dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/trackedentity/AbstractTrackedEntityInstanceService.java

```diff
@@ -456,7 +455,7 @@ public ImportSummary addTrackedEntityInstance( TrackedEntityInstance dtoEntityIn
 
         Set<ImportConflict> importConflicts = new HashSet<>();
         importConflicts.addAll( checkTrackedEntityType( dtoEntityInstance, importOptions ) );
-        importConflicts.addAll( checkAttributes( dtoEntityInstance, importOptions ) );
```

Comment:
very minor 
consider using

`Set<ImportConflict> importConflicts = new HashSet<>(checkTrackedEntityType(dtoEntityInstance, importOptions));`

Reply:
I prefer my version because of readibility. (We discussed on Slack). Not fixing.